easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.2-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,38 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'ConfigureMake'
+
+name = 'FriBidi'
+version = '1.0.2'
+
+homepage = 'https://github.com/fribidi/fribidi'
+
+description = """
+ The Free Implementation of the Unicode Bidirectional Algorithm.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+# https://github.com/fribidi/fribidi/releases/download/v1.0.2/fribidi-1.0.2.tar.bz2
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['bd6d1b530c4f6066f42461200ed6a31f2db8db208570ea4ccaab2b935e88832b']
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('binutils', '2.28'),
+    ('libtool', '2.4.6'),
+]
+
+preconfigopts = """
+    ./autogen.sh
+"""
```

Comment:
@JackPerdue Are you sure this is needed? Worked fine in https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/f/FriBidi/FriBidi-1.0.1-foss-2018a.eb without `autogen.sh`...